### PR TITLE
Fix Dependabot Vulnerability

### PR DESCRIPTION
**Pull request by Kalaiselvi84**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, press enter to put that in a new line, and remove leading whitespace from that line:
>
> /kind breaking
> /kind bug

/kind cleanup

> /kind documentation
> /kind feature
> /kind hotfix
> /kind release

**What this PR does / Why we need it**:

**Which issue(s) this PR fixes**:

Closes #

**Special notes for your reviewer**:

### CODE REVIEW

**Comment by markmandel** [comment]
Found this error:

```
npm WARN tarball tarball data for protobufjs@https://registry.npmjs.org/protobufjs/-/protobufjs-7.2.4.tgz (sha512-4ZPTPkXCdel3+L81yw3dG6+Kq3umdWKh7Dc7GW/CpNk4SX3hK58iPCWeCyhVTDrbkNeKrYNZ7EojM5WDaEWTLQ==) seems to be corrupted. Trying again.
npm WARN tarball tarball data for protobufjs@https://registry.npmjs.org/protobufjs/-/protobufjs-7.2.4.tgz (sha512-4ZPTPkXCdel3+L81yw3dG6+Kq3umdWKh7Dc7GW/CpNk4SX3hK58iPCWeCyhVTDrbkNeKrYNZ7EojM5WDaEWTLQ==) seems to be corrupted. Trying again.
npm ERR! code EINTEGRITY
npm ERR! sha512-4ZPTPkXCdel3+L81yw3dG6+Kq3umdWKh7Dc7GW/CpNk4SX3hK58iPCWeCyhVTDrbkNeKrYNZ7EojM5WDaEWTLQ== integrity checksum failed when using sha512: wanted sha512-4ZPTPkXCdel3+L81yw3dG6+Kq3umdWKh7Dc7GW/CpNk4SX3hK58iPCWeCyhVTDrbkNeKrYNZ7EojM5WDaEWTLQ== but got sha512-AT+RJgD2sH8phPmCf7OUZR8xGdcJRga4+1cOaXJ64hvcSkVhNcRHOwIxUatPH15+nj59WAGTDv3LSGZPEQbJaQ==. (574857 bytes)
```

@steven-supersolid you are our resident NodeJS expect -- is this the right approach to fix this?

**Review by steven-supersolid — APPROVED** [review]

**Comment by steven-supersolid** [comment]
This looks like a transitive dependency so would just try `npm audit fix` which is what I assumed was done but actually probably not if the SHA was not changed - was this just a manual update to package-lock.json and that is why?

To update all dependencies we could try `npm update` which in most cases will work, but should be sure to run tests

We may also want to do a more drastic dependency update using `npm outdated` and then update `package.json` as well but this could require code changes

**Review by steven-supersolid — CHANGES_REQUESTED** [review]
Revoking approval until we clarify how the file was modified and resolve the build issue

**Comment by markmandel** [comment]
> > Revoking approval until we clarify how the file was modified and resolve the build issue
> 
> Initially, I tried with `go get -u github.com/protobufjs/protobuf.js@latest`. This command updated the `go.mod` and `go.sum` files with `github.com/protobufjs/protobuf.js v6.11.3+incompatible`. However, since the [latest version](https://www.npmjs.com/package/protobufjs) of protobufjs is `7.2.4`, I manually updated it.

Since this is a nodejs issue, Go tooling will not work here unfortunately.

If you want a quick node shell with npm tooling - `make sdk-shell-node` will give you one inside the SDK's build image.

File: package-lock.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
```
Comment (steven-supersolid) [inline package-lock.json:1]: This file may have been inadvertently added e.g. by running an npm command in the root directory of the project

Comment (Kalaiselvi84) [reply package-lock.json]: After running `npm update`, the `agones/package-lock.json` file was generated, thought it was required but I have now deleted it. Thanks!

**Comment by markmandel** [comment]
Yay passing. I think this should be better now?

Thanks @steven-supersolid for the review!

**Review by steven-supersolid — APPROVED** [review]

**Review by markmandel — APPROVED** [review]

**Pull request merged**